src/environmentd/src/http/sql.rs

```diff
@@ -830,3 +866,69 @@ fn is_txn_exit_stmt(stmt: &Statement<Raw>) -> bool {
         Statement::Commit(_) | Statement::Rollback(_) | Statement::Prepare(_)
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::WebSocketAuth;
+
+    #[test]
+    fn smoke_test_websocket_auth_parse() {
```

Comment:
Could this be converted to a table-based test so it's easier to extend in the future?

Reply:
Done!